src/sentry/web/frontend/newest_performance_issue.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.auth.superuser import is_active_superuser
+from sentry.models.group import Group, GroupStatus
+from sentry.models.project import Project, ProjectStatus
+
+from .react_page import ReactPageView
+
+
+def get_projects(request, organization):
+    if is_active_superuser(request):
+        return list(
+            Project.objects.filter(status=ProjectStatus.VISIBLE, organization_id=organization.id)
+        )
+    else:
+        return request.access.projects
+
+
+class NewestPerfomanceIssueView(ReactPageView):
+    def handle(self, request: Request, organization, **kwargs) -> Response:
+        projects = get_projects(request, organization)
+        project_ids = [project.id for project in projects]
+
+        group = (
+            Group.objects.filter(
+                project_id__in=project_ids,
+                status=GroupStatus.UNRESOLVED,
+                # performance issue range
+                type__gte=1000,
+                type__lt=2000,
```

Comment:
Is this just temporary? Would prefer to not have these hardcoded if it's long term

Reply:
@wedamija Well, I can use the enum for the lower end but there's nothing on the higher end. What would you recommend?

Reply:
We'll probably provide some constants at some point. But if this is short term then we don't need to worry about updating it

Reply:
@wedamija If this is still around then, just ping me and I'll update it here.